docs/guides/software-lifecycle/security/secrets-detection/README.md

```diff
@@ -40,17 +40,15 @@ To get the most out of `detect-secrets`, you'll need:
 ## Quick Start
 
 1. Install slim-detect-secrets:
-  
-  > ℹ️ **Note:** the SLIM project has customized the Detect Secrets tool to identify additional sensitive keywords such as IP addresses, file paths, and AWS information. These additions are currently [under review](https://github.com/Yelp/detect-secrets/pulls/perryzjc) by the detect-secrets team for merge into the tool's `main` codebase. Until then we recommend using our SLIM fork as described below. 
    
    ```bash
-   pip install git+https://github.com/NASA-AMMOS/slim-detect-secrets.git@exp
+   pip install git+https://github.com/Yelp/detect-secrets.git
    ```
    
 2. Execute a baseline scan:
 
    ```bash
-   detect-secrets scan --all-files --disable-plugin AbsolutePathDetectorExperimental --exclude-files '\.secrets.*' --exclude-files '\.git*' > .secrets.baseline
+   detect-secrets scan --all-files --exclude-files '\.secrets.*' --exclude-files '\.git*' > .secrets.baseline
```

Comment:
The `detect-secrets` tool takes a regexp, not a glob pattern, so

    --exclude-files '\.git*'

means exclude any files that start with a `.` and end with zero or more `t`s, i.e., `.gi`, `.git`, `.gittttt`, etc. What you probably want is

    --exclude-files '\.git.*'

Reply:
Thanks @nutjob4life  - resolved!